### ISSUE TITLE
Variation Selectors?

### ISSUE THREAD

**Issue by hckiang** [stevengj/subsuper-proposal#14]
Have you considered [Variation Selectors](https://en.wikipedia.org/wiki/Variant_form_(Unicode)) might be a solution to this? Currently subscript/superscript can be done in OpenType GSUB look up table (in a non-mathematical setting, I am not talking about the OpenType math table, which is an entirely different thing than GSUB) - this has a disadvantage that the editor/wordprocessors must support OpenType and there is no unicode code point allocated to the subscripts. But the slashed zero might be an example of how stylistic variants can be compatible with both GSUB and Unicode...

If we look up the Wikipedia page of slashed zero, we can see that U+FE00 followed by zero should give us the slashed zero. So there are (at least) two ways to type the slashed zero: either using the OpenType stylistic alternate feature ('ss01', or 'zero'), which requires the application to directly support enabling/disabling these features, or simply having a ligature on 'U+FE00 U+0030' (using 'liga'). The letter approach is more widely compatible because 'liga' feature is enabled by many font rendering engines, so the IDE/GUI developers doesn't need to take care of that, and it will 'just work'.

Now going back to superscripts/subscripts, I can imagine having 'U+FExx U+0065' mapped to 'e subscript', and this solution only requires two additional code points, namely the 'superscript variational selector' and 'subscript variational selector'... What do you think about this?

**Comment by stevengj** [comment]
> The mechanism is intended for variant forms where, generally, if the variant form is unavailable, displaying the base character does not change the meaning of the text, and may not even be noticeable by many readers.

This doesn't seem to apply to mathematical sub/superscripts.

> and this solution only requires two additional code points, namely the 'superscript variational selector' and 'subscript variational selector'

Our current proposal is also for two additional codepoints, just interpreted as "modifier" characters rather than variation selectors.

**Comment by hckiang** [comment]
Good Point! Another thing is that do you think '\in' should also has subscript, or do you think this is too much? It'd be nice to have a variable name like '\sum_{i\in I} x_i'... I'd really want to see this getting approved soon :)
